game-core/src/main/java/games/strategy/triplea/util/WrappedInvocationHandler.java

```diff
@@ -30,9 +30,8 @@ private boolean wrappedEquals(final Object other) {
   }
 
   protected boolean shouldHandle(final Method method, final Object[] args) {
-    if (method.getName().equals("equals") && args != null && args.length == 1) {
-      return true;
-    } else if (method.getName().equals("hashCode") && args == null) {
+    if (method.getName().equals("equals") && args != null && args.length == 1
+        || method.getName().equals("hashCode") && args == null) {
       return true;
     }
     return method.getName().equals("toString") && args == null;
```

Comment:
Parentheses please + this if statement can be combined with the second return statement into a single boolean expression using a logical OR